spigot/src/main/java/com/sekwah/advancedportals/spigot/AdvancedPortalsPlugin.java

```diff
@@ -126,4 +129,13 @@ private void checkAndCreateConfig() {
     public void onDisable() {
         this.portalsCore.onDisable();
     }
+
+    public void registerPlaceholderAPI() {
+        if (this.getServer().getPluginManager().getPlugin("PlaceholderAPI") != null) {
+            AdvancedPortalsCore.getInstance().getTagRegistry().registerTag(new ConditionsTag());
+            getLogger().info("Registered PlaceholderAPI tag");
+            return;
+        }
+        getLogger().info("PlaceholderAPI not found, skipping tag registration");
```

Comment:
Realistically, we won't need to log that it was skipped otherwise we will slowly build up a tonne of logs.

The absence of the registering log on an appropriate plugin version would be proof enough it didn't.